data/json/dreams.json

```diff
@@ -106,7 +106,15 @@
   },
   {
     "type": "dream",
-    "messages": [ "You have a strange dream.", "You feel a yearning..." ],
+    "messages": [
+      "You have a strange dream.",
+      "You have a puckish dream.",
+      "You have a playful dream.",
+      "You have a strange dream about orcs.",
```

Comment:
ELF-A mutants may be sort of tree-people, but they're definitely not Tolkien elves, and don't have anything to do with orcs.